test/test_lang_bind_helper.cpp

```diff
@@ -12184,7 +12184,7 @@ TEST(LangBindHelper_InRealmHistory_Downgrade)
     {
         // In-Realm history
         std::unique_ptr<Replication> hist = make_in_realm_history(path);
-        SharedGroup sg(*hist, SharedGroupOptions(crypt_key()));
```

Comment:
It is a bit unrelated to the rest of this PR, only that I found it while testing my changes here with encryption and keys enabled. The SharedGroup is opened up below without an encryption key, an exception should be thrown for incompatible histories but there would be a different error if one was encrypted and one was not. Equivalently we could introduce encryption on the shared group on line 12193 below (they should both be either encrypted or not encrypted). I think this must have been an oversight when I upgraded the SharedGroup constructor syntax, there was a lot of these to change in this file and I missed this one.